src/js/api/api.js

```diff
@@ -489,21 +347,18 @@ define([
 
             while(_eventQueue.length) {
                 var val = _eventQueue.shift();
-                _eventListener(val[0], val[1]);
+                this.on(val[0], val[1], val[2]);
             }
 
-            // Required to force api to forward events from the controller
-            _eventListener(events.JWPLAYER_MEDIA_SEEKED, utils.noop);
-
-            _eventListener(events.JWPLAYER_PLAYLIST_ITEM, function () {
+            this.on(events.JWPLAYER_PLAYLIST_ITEM, function () {
                 _itemMeta = {};
             });
 
-            _eventListener(events.JWPLAYER_MEDIA_META, function (data) {
+            this.on(events.JWPLAYER_MEDIA_META, function (data) {
                 _.extend(_itemMeta, data.metadata);
```

Comment:
oh didn't realize the js player did this. not at all the same as the Flash player. Not so good...